src/commands/lambda/functions/uninstrument.ts

```diff
@@ -8,6 +8,7 @@ import {
   HANDLER_LOCATION,
   KMS_API_KEY_ENV_VAR,
   LAMBDA_HANDLER_ENV_VAR,
+  LIST_FUNCTIONS_MAX_RETRY_COUNT,
```

Comment:
Do we remove the sourceCodeIntegration flag? Now that the flag is merged it, we should make sure that we're also uninstrumenting it.

Reply:
I just checked, and we are already uninstrumenting it by removing the DD_TAGS where the flags might appear.